traitsui/testing/tester/wx/helpers.py

```diff
@@ -134,6 +227,18 @@ def key_click_text_ctrl(control, interaction, delay):
         wx.MilliSleep(delay)
         control.WriteText(interaction.key)
 
+def get_text_selection_span(control):
+    if isinstance(control, wx.ComboBox):
+        a, b = control.GetTextSelection()
+        if a != b:
+            return (a, b)
+        else:
+            return (-1, -1)
+    else:  # control is wx.TextCtrl
+        if control.GetStringSelection():
+            return control.GetSelection()
+        else:
+            return (-1, -1)
```

Comment:
This fixes the mentioned test failures but it is a very ugly solution.  I could not decide on the best way to make this fix.  @rahulporuri maybe we could have a quick chat to discuss on Monday?